src/bijectors/named_bijector.jl

```diff
@@ -141,8 +141,8 @@ end
     for i = 2:N - 1
         temp = gensym(:res)
         push!(expr.args, :($temp = forward(cb.bs[$i], y)))
```

Comment:
And another `forward` above:
```suggestion
        temp = gensym(:res_logjac)
        push!(expr.args, :(y, $temp = with_logabsdet_jacobian(cb.bs[$i], y)))
```